weasyl/moderation.py

```diff
@@ -561,21 +561,41 @@ def gallery_blacklisted_tags(userid, otherid):
 
 
 def hidesubmission(submitid):
-    d.execute("UPDATE submission SET settings = settings || 'h' WHERE submitid = %i AND settings !~ 'h'", [submitid])
+    d.engine.execute("UPDATE submission SET settings = settings || 'h' WHERE submitid = %(submitid)s AND settings !~ 'h'", submitid=submitid)
```

Comment:
Can you split these unrelated changes out into a separate commit? We can merge them right away.